daal4py/sklearn/linear_model/_ridge_0_22.py

```diff
@@ -107,15 +108,18 @@ def fit(self, X, y, sample_weight=None):
             sample_weight is not None):
         if hasattr(self, 'daal_model_'):
             del self.daal_model_
+        logging.info("sklearn.linear_model.Ridge.fit: " + fit_method_uses_sklearn)
         return super(Ridge, self).fit(X, y, sample_weight=sample_weight)
     else:
         self.n_iter_ = None
         res = _daal4py_fit(self, X, y)
         if res is None:
+            logging.info("sklearn.linear_model.Ridge.fit: " + fit_method_uses_sklearn)
             if hasattr(self, 'daal_model_'):
                 del self.daal_model_
             return super(Ridge, self).fit(X, y, sample_weight=sample_weight)
         else:
+            logging.info("sklearn.linear_model.Ridge.fit: " + fit_method_uses_daal)
```

Comment:
I see that in some cases we print information after DAAL called, and in some cases  - before. I understand why in some cases we can print information only after. But I think it would be better if we did it the same way - everywhere. For example, after a call DAAL. What do you think about it?

Reply:
I think in case of when we go into daal4py, but can't do smth during the running - we need to have 2 prints:
```
"sklearn.linear_model.Ridge.fit: uses Intel® DAAL solver")
"sklearn.linear_model.Ridge.fit: uses original Scikit-learn solver, because the task can't be solved with Intel® DAAL"
```

Reply:
I always print log before call daal4py/sklearn function, but this cases is an exception. Here we are forced to print after training daal4py in the condition.

Reply:
It much better 👍